app/models/person.rb

```diff
@@ -0,0 +1,18 @@
+class Person < ActiveRecord::Base
+  has_many :tasks
+  validates_presence_of :name
+
+  def self.by_name 
+    order(:name)
+  end
+
+  def self.search(search)
+    if search
+      search = search.gsub(/[^A-Za-z]/, '')
+      where("name LIKE ?", "%#{search}%")
+    else
+      order(name: :asc)
+    end
+  end
```

Comment:
You might actually want to move the if/else portion of this search function into the index method in the people_controller file. I was a little confused when I saw that the /people page showed me a list of all the people, but the code was calling a Person.search method instead of People.all or something like that. The way that you implemented it for Tasks is a little clearer for others reading your code!